File: sqrt/sqrt.c
#include<stdio.h>
void main()
{
 float m,n;
 float num;
 n=0.0001;
 printf("ENTER A NUMBER : ");
 scanf("%f",&num);
 for(m=0;m<num;m=m+n)
 {
  if((m*m)>num)
  {
   m=m-n;
   break;
  }
 }
 printf("%.2f",m);
 getch();
 return 1;
}

File: sqrt/sqrt.cs
using System;

namespace GetSquare
{
	class Program
	{
		static void Main(string[] args)
		{
			Console.Write("Enter a number to get the square root: ");
			int n = Int32.Parse(Console.ReadLine());
			int sqr;
			if (n >= 0)
			{
				sqr = getSqr(n);
				Console.WriteLine("The square root of {0} is {1}", n, sqr);
			}

		}

		private static float getSqr(int toSqr)
		{
			return Math.Round(Math.Sqrt(toSqr), 2);
		}
	}
}
